JOSESwift/Sources/ECKeyCodable.swift

```diff
@@ -62,11 +68,16 @@ extension ECPublicKey: Decodable {
         }
 
         // Other common parameters are optional.
-        var parameters: [String: String] = [:]
-        for key in commonParameters.allKeys where !JWKParameter.nonStringParameters.contains(key) {
+        var parameters: [String: JWKParameterType] = [:]
+
+        for key in commonParameters.allKeys where key.type == String.self{
             parameters[key.rawValue] = try commonParameters.decode(String.self, forKey: key)
         }
 
+        for key in commonParameters.allKeys where key.type == [String].self{
```

Comment:
Nitpick: why not merge these two loops and switch on type in the loop body? You do this in a handful of other places in the code as well, but I'll just do the one comment here.

Reply:
Good point! I will definitely update that! 🎖